voice/methods/calls/about.md

```diff
@@ -16,10 +16,5 @@ Read More about Calls and Voice in the <a href="http://dev.bandwidth.com/faq/#vo
 | HTTP Method                        | Path                                                                                            | Description                                                                  |
 |:-----------------------------------|:------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------|
 | <code class="post">post</code>     | [`/calls`](postCalls.md)                                                                        | Create an outbound phone call                                                |
+| <code class="post">GET</code>      | [`/calls/{callId}`](getCallsCallId.md)                                                          | Retrieve call information                                                    |
 | <code class="post">POST</code>     | [`/calls/{callId}`](postCallsCallId.md)                                                         | Replace an active call's BXML                                                |
-| <code class="put">PUT</code>       | [`/calls/{callId}/recording`](putCallsCallIdRecording.md)                                       | Pause or resume a recording on an active call                                |
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings`](getCallsCallIdRecordings.md)                                     | Retrieve information about all of the recordings that occurred during a call |
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings/{recordingId}`](getCallsCallIdRecordingsRecordingId.md)            | Retrieve information about a recording                                       |
-| <code class="delete">DELETE</code> | [`/calls/{callId}/recordings/{recordingId}`](deleteCallsCallIdRecordingsRecordingId.md)         | Delete a recording                                                           |
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings/{recordingId}/media`](getCallsCallIdRecordingsRecordingIdMedia.md) | Retrieve a recording                                                         |
-| <code class="get">GET</code>       | [`/recordings`](getRecordings.md)                                                               | Returns a max of 1000 recordings, sorted by startTime from oldest to newest  |
```

Comment:
The pages pointed by those links were moved to another path and referenced from another page. These links here are pointing to the wrong path and seems they were forgotten.